### ISSUE TITLE
Option to disable auto uploads temporarily

### ISSUE THREAD

**Issue by minecraftchest1** [nextcloud/android#8196]
### Is your feature request related to a problem? Please describe.
Currently, there is no easy way to disable all auto uploads and backup if a users contact book.

### Describe the solution you'd like
A switch to toggle all auto-uploads on or off, as well as contact book backups

### Describe alternatives you've considered
Turning off sync for each folder individually.

### Additional context
I have a nextcloud server running on a raspberrypi4, that is currently turned off as I am migrating to another SD card with a different partitioning setup. I started the migration a week ago, and have not yet setup nextcloud on the new SD card. Therefore, I get a notification saying that backup and sync as failed as it cannot contact the server. I am requesting an option to easily disable these features temporarily. 

This feature would also benefit users who are going off of the network for a while and don't want these messages.

 **NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by joshtrichards** [comment]
Duplicate of #8393

**Comment by ernolf** [comment]
> Duplicate of #8393

I disagree. This is not a duplicate. This is about the (missing) ability to disable the auto upload feature and not about a feature to stop all transfers (temporarily)

ernolf

Some context:
https://help.nextcloud.com/t/instant-upload-disable/192725

**Comment by PaulMcF1987** [comment]
I agree with @ernolf 
And with the original post

The auto upload function should be optional and not forced on the end user as per my nextcould support forum question -https://help.nextcloud.com/t/instant-upload-disable/192725/2

I use nextcloud purely for work purposes and do not want my personal things upl9aded from my phone
There should he the ability to turn off and/or chose what gets uploaded automatically

**Comment by spacedrone404** [comment]
+1 autoupload should be disabled by default and must have toggle switch.

**Comment by joshtrichards** [comment]
> There should he the ability to turn off and/or chose what gets uploaded automatically


> autoupload should be disabled by default and must have toggle switch.

To be clear, the auto upload feature is off by default. It does not upload anything unless explicitly turned on for a given folder.

**Comment by LuisAPI** [comment]
I highly agree with OP’s request. I think there should be an easy way to disable all auto-uploads configured with a single tap.

I made some quick UI mock-ups of the Manage Auto-Upload screen below, using the Japanese-era pause ⏸️ and repeat 🔁 emojis which I would suppose would be best placed at the top right, right next to the triple-dotted More button.

|MAU with Pause|MAU with Restart|
|:-:|:-:|
|![Screenshot of “Manage auto–upload” screen with inserted Pause (⏸️) button in top header](https://github.com/user-attachments/assets/394781f4-5208-410b-a1cf-1b62d48ca479) | ![Screenshot of “Manage auto–upload” screen with inserted Repeat (🔁) button in top header](https://github.com/user-attachments/assets/289f0e57-77d8-42af-9f77-8c9f6a09d61e) |


-----

> I think the rest of you are possibly concerned about something else: the notifications about new media folders being detected which _offer_ to configure auto upload if you wish for them.

Admittedly off-topic, but is there an existing issue filed that I can thumbs-up as well regarding this? These notifs are incredibly annoying and are instantly pushed to my device the moment I move a photo from one folder to another on either my internal or my microSD storage.

**Comment by spacedrone404** [comment]
just one toggle, on/off.
current state is extremely annoying and not transperent for understanding.
